music_assistant/controllers/streams.py

```diff
@@ -56,6 +57,18 @@ def __init__(self, mass: MusicAssistant):
     @property
     def base_url(self) -> str:
         """Return the base url for the stream engine."""
+
+        """
+        This is a purpously undocumented feature to override the automatic
+        generated base_url used by the streaming-devices.
+
+        If you need this, you know it, but you should probably try to not set it!
```

Comment:
haha!

Reply:
Can you change the comment just to be inline and leave the function docstring alone ?

Reply:
Just the way you wanted it... ;D